chcemvediet/apps/inforequests/templates/inforequests/tables/inforequests.html

```diff
@@ -31,15 +31,32 @@
 {% endblock %}
 
 {% block date %}
-  FIXME
+  {% if object.closed %}
+    {{ object.last_action.legal_date|date:"j. n. Y" }}
+  {% elif object.has_undecided_emails %}
+    {{ object.newest_undecided_email.processed|date:"j. n. Y" }}
+  {% else %}
+    {{ object.last_action.legal_date|date:"j. n. Y" }}
+  {% endif %}
 {% endblock %}
 
 {% block status %}
-  {% if object.has_undecided_emails %}
+  {% if object.closed %}
+    <i class="chv-icon chv-icon-li icon-ok"></i>
+    {{ object.last_action.get_extended_type_display }}
+    {% if object.disclosure_level == ACTION_DISCLOSURE_LEVELS.FULL %}
+      <p><em>{% trans 'inforequests:mine:status:disclosure_level:FULL' %}</em></p>
+    {% elif object.disclosure_level == ACTION_DISCLOSURE_LEVELS.PARTIAL %}
+      <p><em>{% trans 'inforequests:mine:status:disclosure_level:PARTIAL' %}</em></p>
```

Comment:
Pls nepouzivaj `<p>` vo vnutri tabulky. Lebo to vytvara zbytocny vertikalny margin:

![tabulka](https://user-images.githubusercontent.com/5547275/72544383-8b7a2600-3887-11ea-9973-b899d1bb24df.png)

Pozi iba `<br>` na zalomenie.